test/testHelpers/collections/collectionHelper.ts

```diff
@@ -31,3 +39,20 @@ export const createCollectionPayload = (): CollectionPayload => {
   }
   return collectionPayload
 }
+
+export async function createCollectionViaApi(collectionAlias: string): Promise<void> {
```

Comment:
It doesn't seem to be returning void, since the create endpoint when used in the repository it returns CollectionPayload. I think we should use CollectionPayload as the returned type